### ISSUE TITLE
Best way to set a floating windows always on top?

### ISSUE THREAD

**Issue by spadino** [fuhsjr00/bug.n#183]
There's a way to have a windows bet set to float AND always on top from Config.ini?

**Comment by ertwro** [comment]
Is there any progress with this @joten? I'm doing a tutorial for bug.n on YouTube and I wanted to show how to do this by a simple hotkey addition <kbd>Win</kbd><kbd>n</kbd>
```
Config_hotkey=#n::WinSet, AlwaysOnTop, toggle, A
```
But I can't seem to make it work without loading an `ahk` script

**Comment by joten** [comment]
Sorry, no prgress on this issue.
Calling `WinSet` from Config.ini is not implemented, only `Run`, `Send` and bug.n functions.

Viewing multiple tags is not possible in bug.n, only multiple tags for a window.

**Comment by ertwro** [comment]
Oh, geez. I missed that in Usage.md. Sorry. Is there a way to run Run and send in the same line for a hotkey? e.g. 
```
run ms-settings:sound
sleep, 1000
send {Tab 6}
send {Space}
```

**Comment by joten** [comment]
No, currently only one command can be send per Config.ini hotkey. You would have to write a function and associate this with a hotkey, editing Config.ahk and running bug.n as a AutoHotkey script.
